### DISCUSSION TITLE
NOAA Enable features lost

### DISCUSSION THREAD

**Discussion by disgusdad** [egzumer/uv-k5-firmware-custom#490] [Q&A]
Is there any other egzumer features/enhancements that get disabled or don't work other than the scan-list shortcut if compiled with enable_NOAA?

Actually it seems just enabling NOAA fails due to overflow in FLASH section, so I assume I would have to find features to disable to have enough room to compile with that enabled.

**Comment by acbits** [comment]
I had to disable the spectrum analyzer(great feature!) and enable NOAA. The flash image size is 91% and I was able to flash it.

I have set it NOAA frequency 162.400 MHz and it is receiving something. The green LED is flashing and it is displaying DTMF codes on screen. Not sure whether SAME codes are displayed as DTMF or NOAA broadcasts SAME & DTMF codes? 🤔

P.S. While building, you have to do a 
`git checkout v0.22`
before making changes to Makefile as there are some changes between v0.22 and main.
